client/my-sites/stats/controller.jsx

```diff
@@ -639,3 +639,10 @@ export function emailSummary( context, next ) {
 
 	next();
 }
+
+export function participation( context, next ) {
+	context.primary = (
+		<AsyncLoad require="calypso/my-sites/stats/stats-participation" placeholder={ PageLoading } />
+	);
+	next();
+}
```

Comment:
```suggestion
export function purchase( context, next ) {
	context.primary = (
		<AsyncLoad require="calypso/my-sites/stats/purchase" placeholder={ PageLoading } />
	);
	next();
}
```

Maybe "purchase" would be better than "participation"? We're already logging `Stats > Purchase` as the page title for view tracking.